packages/terra-form-validation/src/FormValidationUtil.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable no-restricted-globals */
+
+class FormValidationUtil {
+  /* Checks given value's length against maximum. Returns true if value's length is less than the maximum.
+   * @value is the value string, @maximum is a nonnegative whole number. */
+  static isUnderMaxLength(value, maximum) {
+    if (!(typeof value === 'string' || value instanceof String)
+    || !(typeof maximum === 'number' && isFinite(maximum))
+    || maximum < 0 || !Number.isInteger(maximum)) { return false; }
+    return value.length <= maximum;
+  }
+
+  /* Checks given value's length against minimum. Returns true if value's length is greater than the minimum.
+   * @value is the value string, @minimum is a nonnegative whole number. */
+  static isOverMinLength(value, minimum) {
+    if (!(typeof value === 'string' || value instanceof String)
+    || !(typeof minimum === 'number' && isFinite(minimum))
+    || minimum < 0 || !Number.isInteger(minimum)) { return false; }
+    return value.length >= minimum;
+  }
+
+  /* Checks given value for extraneous whitespace. return true if there is no whitespace.
+   * @value is the value string */
+  static hasNoWhitespace(value) {
+    if (!(typeof value === 'string' || value instanceof String)) { return false; }
+    return value.trim() === value;
+  }
+
+  /* Checks given value for any string in given array. return true if value does not contain any of them.
+   * @value is the value string, @chars is an array of strings  */
+  static containsCharacters(value, chars) {
+    if (!(typeof value === 'string' || value instanceof String)
+    || !(chars && typeof chars === 'object' && chars.constructor === Array)) { return false; }
+    for (let i = 0; i < chars.length; i += 1) {
+      if (value.includes(chars[i])) {
+        return false;
```

Comment:
shouldn't we flip the return statements?  this function is called containsCharacters, so i'd expect it to return true if the value contains one of the strings, not false

Reply:
Yep point, fixed here https://github.com/cerner/terra-framework/pull/635/commits/415c46ab9a0b74893a686c38430b1d8decd708ad#diff-e0dd485bcf22c1ac58c3d5cfd0103965R37, and swapped the jest tests around.